.github/workflows/submit.yml

```diff
@@ -105,13 +105,13 @@ jobs:
         flavor:
           - build release
           - build debug
-          - build hotspot no-pch
+          - build debug hotspot no-pch
```

Comment:
```suggestion
          - build hotspot no-pch
```

Reply:
Right, we can do that. Let me regenerate both #546 and #547